packages/lesswrong/lib/crud/withMulti.ts

```diff
@@ -303,6 +303,7 @@ export function useMulti<
 
   const defaultLimit = ((locationQuery && queryLimitName && parseInt(locationQuery[queryLimitName])) || (terms && terms.limit) || initialLimit)
   const [ limit, setLimit ] = useState(defaultLimit);
+  const [ lastTerms, setLastTerms ] = useState(_.clone(terms));
```

Comment:
The changes in this file were done with Jim - he helped me update `useMulti()` to work properly with filters